lib/guardian/token/paseto.ex

```diff
@@ -56,6 +56,7 @@ defmodule Guardian.Token.Paseto do
           | Guardian.Token.secret_error()
   def create_token(mod, claims, opts) do
     key = secret_key(mod, opts)
+    claims = Poison.encode!(claims)
```

Comment:
This shouldn't rebind to `claims`. Could you please either rename this variable or move the `Poison.encode!/1` call into the `do_create_token/4` call below